pkg/assembler/clients/operations/license.graphql

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2023 The GUAC Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: This is experimental and might change in the future!
+
+# Ingest License
+
+mutation IngestLicense($license: LicenseInputSpec!) {
+  ingestLicense(license: $license) {
+    ...AllLicenseTree
+  }
+}
+
+# Bulk Ingest Licenses
+
+mutation IngestLicenses($licenses: [LicenseInputSpec!]!) {
+  ingestLicenses(licenses: $licenses) {
+    ...AllLicenseTree
```

Comment:
return nothing here once mutation is changed to IDs

Reply:
Done